templates/ingress-nginx/cloud-generic.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Service
```

Comment:
Could we add a comment with the source for this file? I'm wondering if we got it somewhere as-is or came up with it ourselves for use in Ansible?